src/android/CHIPTool/README.md

```diff
@@ -6,3 +6,63 @@ a CHIP device
 
 -   Connection to the chip device's soft AP will be a manual step until pairing
     is implemented.
+
+===========================================
+
+Steps to build CHIP for Android apps
+
+Pre-conditions: Have Android NDK downloaded to your machine. Set $ANDROID_NDK_HOME environment variable to point to the NDK package is downloaded.
+
+1. Checkout the CHIP repo
+
+2. In commandline / Terminal, 'cd' into the top CHIP directory
+
+3. May have to run 'autoconf' / 'autoreconf', if changes to configure.ac have been made.
```

Comment:
This should be `./bootstrap -w config` or just `./bootstrap`.

Reply:
Done.